vcx/ci/Jenkinsfile

```diff
@@ -86,6 +86,12 @@ def mainUbuntu() {
                     sh 'vcx/wrappers/python3/ci/test.sh'
                 }
             }
+            stage('Test Java Wrapper'){
+                java = docker.build("java", "--build-arg uid=${uid} -f ${javaDockerfile} .")
+                java.inside() {
+                    sh 'cd vcx/wrappers/java && ./gradlew test'
```

Comment:
--no-daemon should be added to the ./gradlew test command. We have seen issues in the pipeline where the daemons were persisting on the machines and breaking the build.